models/invites.ts

```diff
@@ -0,0 +1,47 @@
+import firestore from "../utils/firestore";
+import { InviteBody, Invite } from "../types/invites";
+const inviteModel = firestore.collection("invite");
+
```

Comment:
Let's not create a new collection for it, We should use existing collection for discord invites

Reply:
Fix